quote.cpp

```diff
@@ -0,0 +1,44 @@
+#include <string>
+
+namespace Sass {
+  using namespace std;
+
+  string unquote(const string& s)
+  {
+    if (s.empty()) return "";
+    if (s.length() == 1) {
+      if (s[0] == '"' || s[0] == '\'') return "";
+    }
+    char q;
+    if      (*s.begin() == '"'  && *s.rbegin() == '"')  q = '"';
+    else if (*s.begin() == '\'' && *s.rbegin() == '\'') q = '\'';
```

Comment:
Oh nice. I was just checking appveyor build: https://ci.appveyor.com/project/sass/libsass/build/1.0.6. Not sure why mingw made it a matter of fatality and terminated tests for such a trivial reason (it should just be a compiler warning). msvc is also choking with unresolved symbol.